openmmtools/mcmc.py

```diff
@@ -378,16 +378,12 @@ def __iter__(self):
         return iter(self.move_list)
 
     def __getstate__(self):
-        serialization = dict()
-        for i, move in enumerate(self.move_list):
-            serialization['move' + str(i)] = utils.serialize(move)
-        return serialization
+        serialized_moves = [utils.serialize(move) for move in self.move_list]
```

Comment:
Yeah, `__getstate__` should return a dictionary representation of the object. That's the signature for all `__getstate__` methods. The key must be a constant, otherwise `__setstate__` doesn't know what to read.